uberfire-structure/uberfire-structure-backend/src/main/filtered-resources/DependencyTypesMapper.json

```diff
@@ -0,0 +1,38 @@
+{
+  "kieVersion":"${version.org.kie}",
+  "internalArtifacts":[
+    {"groupId":"org.kie", "artifactId":"kie-internal", "scope":"provided"},
+    {"groupId":"org.optaplanner", "artifactId":"optaplanner-core", "scope":"provided"},
+    {"groupId":"org.dashbuilder", "artifactId":"kie-internal", "scope":"provided"}
+  ],
+  "dependencies":[
+    {
+      "type": "JPA",
+      "deps": [
+        {"groupId": "org.hibernate.javax.persistence", "artifactId": "hibernate-jpa-2.1-api", "version": "${version.org.hibernate.javax.persistence.hibernate-jpa-2.1-api}", "scope": "compile"}
```

Comment:
@desmax74 Please reach out to @mbiarnes  he is updating hibernate on master so it would be nice to align it here too

Reply:
@desmax74 I think that new version has different groupId